lib/ruby-debugger.coffee

```diff
@@ -8,18 +8,20 @@ class RubyDebugger
     @context = new DebuggerContext()
     @view = new View(state.viewState, @context)
     @panel = atom.workspace.addRightPanel(item: @view.getElement(), visible: false)
-
+    #TODO start rdebugger
```

Comment:
would be nice to keep track of TODOs/feature ideas another way - ie. Github issues for example.